### PR TITLE
Add changelog generator

### PR DESCRIPTION
**Pull request by srirambv**
Fixes: #99

### CODE REVIEW

**Comment by srirambv** [comment]
ready for re-review

File: changelog.py [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+
+            labels = issue.get_labels()
+            label_names = []
+            for label in labels:
+                label_names.append(label.name)
+            if "crash" in label_names and "tests" not in label_names:
+                crash_line = " - " +\
+                    issue_title + ".(#" + str(issue.number) + ")"
+                crash_list.append(crash_line)
+            if "crash" not in label_names and "webcompat" not in label_names\
```
Comment (bbondy) [inline changelog.py]: A nice way to clear up trailing slashes is just to have an opening parenthesis, then you can do multiple lines until you close it.

Comment (bbondy) [reply changelog.py]: without the need of a slash at the end of each line.

**Comment by srirambv** [comment]
> Maybe this should only consider release-notes/include and error or warn if there are some that are not labeled with include or exclude?

I'll do a follow up for that. Merging this for now

**Pull request merged**